Sources/FaunaDB/JSON.swift

```diff
@@ -44,6 +44,20 @@ internal enum JsonType {
     case null
 }
 
+extension JsonType: Equatable {
+    public static func == (lhs: JsonType, rhs: JsonType) -> Bool {
+        switch (lhs, rhs) {
+        case let (.object(a), .object(b)):   return a == b
+        case let (.array(a), .array(b)):     return a == b
+        case let (.string(a), .string(b)):   return a == b
+        case let (.number(a), .number(b)):   return a == b
+        case let (.double(a), .double(b)):   return a == b
+        case let (.boolean(a), .boolean(b)): return a == b
+        default: return false
```

Comment:
Let's not have a default for enums. If we remove it, the compiler requires exhaustive implementation.